satellite/nodestats/endpoint.go

```diff
@@ -17,8 +19,8 @@ import (
 )
 
 var (
-	// NodeStatsEndpointErr is endpoint error class
-	NodeStatsEndpointErr = errs.Class("node stats endpoint error")
+	// Error is endpoint error class
+	Error = errs.Class("nodestats endpoint error")
```

Comment:
It isn't used anymore, shouldn't we remove it?

Reply:
fixed